addons/website_event/static/src/xml/customize_options.xml

```diff
@@ -3,13 +3,11 @@
 
     <t t-name="website_event.customize_options" id="event_options">
         <li class="dropdown-header">Event Specific</li>
-        <a href="#" name="display-website-menu" class="dropdown-item" role="menuitem">
-            <label class="o_switch" for="display-website-menu">
-                <input id="display-website-menu" type="checkbox"/>
-                <span/>
-                Event Sub-menu
-            </label>
-        </a>
+        <label class="o_switch dropdown-item m-0" for="display-website-menu">
+            <input id="display-website-menu" type="checkbox"/>
+            <span/>
+            Event Sub-menu
+        </label>
```

Comment:
You should also change the inherit part in website_event_meet that used the previous `<a>` to display the "Allow Room Creation" option

Reply:
Hello @jeh-odoo,
I have updated the code. Thanks!